vcpkg.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "ni-media",
+  "version-string": "1.0.0",
```

Comment:
Note: we don't have any tagged releases - maybe this is something else we should consider doing now.

Reply:
This vcpkg version is just a placeholder atm, but yes this might be confusing. I aligned it with version 0.1.0 we have in the cmake project. I am all up for creating a tagged release, let's do that once the CI pipeline is fully in place.